resources/views/layouts/pagination.blade.php

```diff
@@ -36,7 +36,7 @@ class="btn btn-sm btn-link dropdown-toggle p-0 m-0"
                         ->except(['page','_token']))
                         ->links('platform::partials.pagination')
                 !!}
-            @else
+            @elseif($paginator instanceof \Illuminate\Contracts\Pagination\Paginator)
```

Comment:
This is not related to permission checks

Reply:
Oops sorry, I'll fix it as soon as I get home.

Reply:
Done!